crates/biome_css_semantic/src/semantic_model/mod.rs

```diff
@@ -30,248 +30,248 @@ pub fn semantic_model(root: &CssRoot) -> SemanticModel {
-        assert_eq!(rule.declarations.len(), 2);
-
-        assert_eq!(rule.selectors[0].name, "p");
-        assert_eq!(rule.declarations[0].property.name, "color");
-        assert_eq!(rule.declarations[0].value.text, "red");
-
-        assert_eq!(rule.declarations[1].property.name, "font-size");
-        assert_eq!(rule.declarations[1].value.text, "12px");
-
-        let range = TextRange::new(0.into(), 1.into());
-        let rule = model.get_rule_by_range(range).unwrap();
-
-        assert_eq!(rule.selectors.len(), 1);
-        assert_eq!(rule.declarations.len(), 2);
-
-        assert_eq!(rule.selectors[0].name, "p");
-        assert_eq!(rule.declarations[0].property.name, "color");
-        assert_eq!(rule.declarations[0].value.text, "red");
-
-        assert_eq!(rule.declarations[1].property.name, "font-size");
-        assert_eq!(rule.declarations[1].value.text, "12px");
-    }
-
-    #[test]
-    fn test_nested_get_rule_by_range() {
-        let parse = parse_css(
-            r#"p { --foo: red; font-size: 12px;
-            .child { color: var(--foo)}
-            }"#,
-            CssParserOptions::default(),
-        );
-        let root = parse.tree();
-        let model = super::semantic_model(&root);
-
-        // range of the declaration 'blue' in '.child'
-        let range = TextRange::new(60.into(), 64.into());
-        let rule = model.get_rule_by_range(range).unwrap();
-
-        assert_eq!(rule.selectors.len(), 1);
-        assert_eq!(rule.declarations.len(), 1);
-
-        assert_eq!(rule.selectors[0].name, ".child");
-
-        assert_eq!(rule.declarations[0].property.name, "color");
-        assert_eq!(rule.declarations[0].value.text, "var(--foo)");
-
-        let parent = model.get_rule_by_id(rule.parent_id.unwrap()).unwrap();
-        assert_eq!(parent.selectors.len(), 1);
-        assert_eq!(parent.declarations.len(), 2);
-
-        assert_eq!(parent.selectors[0].name, "p");
-        assert_eq!(parent.declarations[0].property.name, "--foo");
-        assert_eq!(parent.declarations[0].value.text, "red");
-
-        assert_eq!(parent.declarations[1].property.name, "font-size");
-        assert_eq!(parent.declarations[1].value.text, "12px");
-    }
-
-    #[ignore]
-    #[test]
-    fn quick_test() {
-        let parse = parse_css(
-            r#"@property --item-size {
-  syntax: "<percentage>";
-  inherits: true;
-  initial-value: 40%;
-}"#,
-            CssParserOptions::default(),
-        );
-
-        let root = parse.tree();
-        let model = super::semantic_model(&root);
-        dbg!(&model.rules());
-        dbg!(&model.global_custom_variables());
-    }
-}
+// #[cfg(test)]
+// mod tests {
+//     use biome_css_parser::parse_css;
+//     use biome_css_parser::CssParserOptions;
+//     use biome_rowan::TextRange;
+
+//     #[test]
+//     fn test_simple_ruleset() {
+//         let parse = parse_css(
+//             r#"p {
+//   font-family: verdana;
+//   font-size: 20px;
+// }"#,
```

Comment:
Temporarily disable the tests. I will make another PR to split them into another module.